airflow/www/static/js/dag/details/taskInstance/MappedInstances.tsx

```diff
@@ -47,7 +47,12 @@ const MappedInstances = ({ dagId, runId, taskId, onRowClicked }: Props) => {
   const sort = sortBy[0];
 
   const orderBy =
-    sort && (sort.id === "state" || sort.id === "mapIndex")
+    sort &&
+    (sort.id === "state" ||
+      sort.id === "mapIndex" ||
+      sort.id === "duration" ||
+      sort.id === "startDate" ||
+      sort.id === "endDate")
```

Comment:
```suggestion
    sort && sort.id
```
We can simplify this check now.